sourcemash/api/feeds.py

```diff
@@ -18,14 +20,75 @@
 
 class FeedListAPI(Resource):
```

Comment:
You don't want to put a DELETE on a list, since that route doesn't specify which feed you're trying to delete. Since we don't have any other admin commands yet--and deleting inappropriate feeds is more of an enhancement--I think we can take time to consider how we handle our admin API and create an "admin permissions" task for the future.

Also remember we don't even have ways of assigning people the admin role yet.